packages/gestalt/src/Tabs.js

```diff
@@ -25,54 +52,68 @@ export default function Tabs({
     text: React.Node,
   |}>,
   wrap?: boolean,
-|}): React.Node {
-  const handleTabClick = (i: number, e: SyntheticMouseEvent<>) =>
-    onChange({ activeTabIndex: i, event: e });
+|};
 
+export default function Tabs({
+  activeTabIndex,
+  onChange,
+  size = 'md',
+  tabs,
+  wrap,
+}: Props): React.Node {
   return (
-    <div
-      className={classnames(
-        styles.Tabs,
-        wrap && layout.flexWrap,
-        size === 'md' ? layout.medium : layout.large
-      )}
+    <Box
+      alignItems="center"
+      display="flex"
+      marginEnd={-1}
+      marginStart={-1}
+      minHeight={size === 'lg' ? 48 : 40}
       role="tablist"
+      wrap={wrap}
+      dangerouslySetInlineStyle={{
+        __style: { userSelect: 'none' },
+      }}
     >
-      {tabs.map(({ href, id, text }, i) => {
-        const isActive = i === activeTabIndex;
-        const cs = classnames(styles.tab, {
-          [styles.tabIsNotActive]: !isActive,
-          [styles.tabIsActive]: isActive,
-        });
+      {tabs.map(({ id, href, text }, index) => {
+        const isActive = activeTabIndex === index;
         return (
-          <a
-            aria-selected={isActive}
-            className={cs}
-            href={href}
-            {...(id ? { id } : {})}
-            key={`${i}${href}`}
-            onClick={(e: SyntheticMouseEvent<>) => handleTabClick(i, e)}
-            role="tab"
+          <Box
+            key={id || `${href}_${index}`}
+            id={id}
+            dangerouslySetInlineStyle={{
+              __style: { padding: '0px 2px' },
+            }}
           >
-            <Text color={isActive ? 'white' : 'darkGray'} weight="bold">
-              {text}
-            </Text>
-          </a>
+            <Link
+              accessibilitySelected={isActive}
+              hoverStyle="none"
+              href={href}
+              onClick={({ event }) => {
+                event.preventDefault();
```

Comment:
@jackhsu978 could we avoid the `preventDefault` here? Ideally it's something the consumer of `<Tabs />` does.

Main use case I'm thinking of is `CMD+Click` to open the `<Tab />` in a new browser tab. If the consumer does an extra check for CMD+Click and only does `preventDefault` then, we should be good.

Reply:
you're right! thanks for catching this!